spec/vcr/applications_controller/edit/non_lease_up_application.yml

```diff
@@ -21,7 +21,7 @@ http_interactions:
       message: OK
     headers:
       Date:
-      - Fri, 01 Nov 2019 21:38:48 GMT
```

Comment:
all of these cassette changes are because moving the listing service queries to the other file ended up adding more spaces to the urls. I thought about trying to figure out a way to ignore it, but there wasn't one that was immediately apparent so I updated the cassettes instead.